lib/esformatter.js

```diff
@@ -41,16 +41,19 @@ exports.unregister = plugins.unregister;
 
 
 function format(str, opts) {
+  _options.set(opts);
   str = plugins.stringBefore(str);
   var ast = rocambole.parse(str);
-  transform(ast, opts);
+  transform(ast, opts, true /*options already set, so skip setting there here*/);
```

Comment:
uhm.. even tho this is the easiest solution I don't think we should add a flag to `transform`; it will end up biting us down the road..